editor/import/editor_import_collada.cpp

```diff
@@ -490,120 +490,6 @@ Error ColladaImport::_create_material(const String &p_target) {
 	return OK;
 }
 
-static void _generate_normals(const PoolVector<int> &p_indices, const PoolVector<Vector3> &p_vertices, PoolVector<Vector3> &r_normals) {
```

Comment:
For the reference, this was no longer in use since a2027cfa49610cf7653288ddaffdbe33c70e75f8 which refactored normals generation with `SurfaceTool`.